### ISSUE TITLE
Maps styling conflicts

### ISSUE THREAD

**Issue by woowoowoo** [unimelb/pattern-lib#1234]
I need a little help resolving two problems that are occurring on maps.unimelb

1) Strange header (logo) on directions page. https://maps.unimelb.edu.au/dir?assetB=29004
This page is a special page, with just minimal branding, so I've almost certainly stuffed up somewhere, but I can't find where. Probably need someone to go through my matrix workings with me.

2) On all other maps pages (eg: https://maps.unimelb.edu.au/) clicking the default Search button results in a mind-bending rotation animation of `<div class="blanket"></div>`. If I disable the MazeMap css file, the problem disappears. I suspect it's going to be easier to fix this in our code than to get MazeMap to apply a fix to theirs.

**Comment by TimMansell** [comment]
> I need a little help resolving two problems that are occurring on maps.unimelb
> 
> 1. Strange header (logo) on directions page. https://maps.unimelb.edu.au/dir?assetB=29004
>    This page is a special page, with just minimal branding, so I've almost certainly stuffed up somewhere, but I can't find where. Probably need someone to go through my matrix workings with me.

A fix to the logo was release in v4.10.0 of pattern library.  Can you update to that (or even better the version we're running at the moment v4.14.0)

You'll also need to update the logo image to use the new padded svg.

```html
<img alt="The University of Melbourne homepage" src="https://d2glwx35mhbfwf.cloudfront.net/v4.15.0/logo-with-padding.svg" class="logo__image logo__image--md logo--no-padding-md">
```

**Comment by TimMansell** [comment]
> 2\. in a mind-bending rotation animation of `<div class="blanket"></div>`. If I disable the MazeMap css file, the problem disappears. I suspect it's going to be easier to fix this in our code than to get MazeMap to apply a fix to theirs.

will investigate this

**Comment by TimMansell** [comment]
I have created a PR for a fix to #2. It will be available in next release.

https://github.com/unimelb/pattern-lib/pull/1248

**Comment by TimMansell** [comment]
This fix has been released as part of patter lib 4.16.0
